### PR TITLE
added name purujit

### PR DESCRIPTION
**Pull request by mockingmoon**
hi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,19 @@
...
+
+
+
+
+
+
+
+
+
+  - Purujit S
```
Comment (meta-boy) [inline README.md:20]: You might want to tone it down with the newline

**Comment by mockingmoon** [comment]
removed newliines

**Comment by meta-boy** [comment]
You do not need to close a pull request every time you fix your changes as you can just push the commit with the fix.

**Comment by mockingmoon** [comment]
how to pull latest version? it says already up to date

**Comment by mockingmoon** [comment]
done

**Pull request merged**